### DISCUSSION TITLE
给出的预览PDF里面的摘要和正文的字体不一样是怎么实现的呀

### DISCUSSION THREAD

**Discussion by Sia-xinghao** [Keldos-Li/typora-latex-theme#127] [General]
rt，摘要是楷体，正文是宋体这种效果如何实现

**Comment by Keldos-Li** [comment]
你可以查看我们的样例文件，这在release中也会提供。
- https://github.com/Keldos-Li/typora-latex-theme/blob/main/resources/essay-template.md?plain=1

**Reply by Sia-xinghao** [reply]
了解了，原来是用HTML写的，非常感谢！
